src/log-filestore.c

```diff
@@ -22,6 +22,9 @@
  *
  */
 
+#include <sys/types.h>
```

Comment:
these should be unnecessary as they are included in suricata-common.h, so please remove them here